openlcsd/flow/tasks.py

```diff
@@ -919,12 +919,11 @@ def get_container_remote_source(context, engine):
             type_components = components.get(comp_type)
             if type_components:
                 rs_components.extend(type_components)
-        missing_components = sc_handler.get_container_remote_source(
-            rs_components)
+        missing_components, missing_components_errors = \
+            sc_handler.get_container_remote_source(rs_components)
         if missing_components:
-            engine.logger.error('Failed to get remote source for components:')
-            for missing_component in missing_components:
-                engine.logger.error(missing_component)
+            for misc_error in missing_components_errors:
+                engine.logger.error(misc_error)
```

Comment:
use `join` instead of forloop? also I think you mean `missing` instead of `misc`.

Reply:
Updated